src/main/java/com/jfrog/ide/idea/inspections/JFrogSecurityWarning.java

```diff
@@ -62,7 +62,7 @@ public JFrogSecurityWarning(SarifResult result, SourceCodeScanType reporter) {
                 result.getRuleId(),
                 getFirstRegion(result).getSnippet().getText(),
                 reporter,
-                !result.getKind().equals("pass"),
+                (!result.getKind().equals("pass") && (rule.getRuleProperties().map(properties -> properties.getApplicability().equals("applicable")).orElse(true))),
```

Comment:
On one side, this is very long
problem is this is the super, meaning I cant use a private func and call it. 
I can put a defualt value and edit, but that is gross

Reply:
why not creating a value before it that'll calculate it?
having a map, and getting into and a property of a property of a.... 
isn't a best-practice..

And while you're at it - do the same for line 61

Reply:
um okay.
I still dont love it . but its an okay alternative

Reply:
i just made them static instead